pkg/aws/apis/aws_provider_spec.go

```diff
@@ -13,7 +13,7 @@ limitations under the License.
 
 package api
 
-//AWSProviderSpec is the spec to be used while parsing the gRPC calls.
+//AWSProviderSpec is the spec to be used while parsing the calls.
```

Comment:
are we using `type Secrets struct` defined in this file anymore? If not, can we clean it up?

Reply:
Yes nice catch. Have removed them with this commit - https://github.com/gardener/machine-controller-manager-provider-aws/pull/10/commits/de39f3f87c093d4f442677de5282f0412fce224b